docs/modules/reference/pages/service-assurance/monitors/WmiMonitor.adoc

```diff
@@ -0,0 +1,120 @@
+
+= WmiMonitor
+
+The WMI monitor tests the availability of WMI services on remote interfaces.
+
+NOTE: xref:reference:service-assurance/monitors/WsManMonitor.adoc[WSMAN] does everything that WMI does and is a more modern protocol for Windows monitoring.
+
+== Monitor facts
+
+[cols="1,7"]
+|===
+| Class Name
+| `org.opennms.netmgt.poller.monitors.WmiMonitor`
+|===
+
+== Configuration and use
+
+Note that username, password, and domain credentials defined here override any defined in the wmi-config file.
```

Comment:
@Bonrob2 Is it required to have the credentials within the wmi-config file? If not MetaData's are also an option to avoid credentials in poller configs. But maybe this is a general hint we could add in the common section.

Reply:
It's not required, though if using directory credentials, using the config file can apply a single credential to a range of devices rather than storing individual credentials in metadata and/or service configs.